docs/6.x/faq.md

```diff
@@ -104,6 +104,20 @@ if [ ! -z "$timesync_bus_name" ]; then
 fi
 ```
 
+## VMWare ESXi
+
+On virtual machines powered by VMWare ESXi (part of VMWare vSphere suite) port
```

Comment:
I think this only occurs on certain versions or certain configurations of vmware, which might be worth linking to. I think @eldios or @UnderGreen may have found the specific docs / version at some point, which might be worth mentioning and/or linking to.

Reply:
@knisbet That's a good point. @eldios / @UnderGreen - do you guys recall where you found the said link?

Reply:
https://docs.vmware.com/en/VMware-NSX-Data-Center-for-vSphere/6.4/com.vmware.nsx.install.doc/GUID-E7C4E61C-1F36-457C-ACC5-EAF955C46E8B.html
VXLAN is part of VMware NSX. `8472 (default before NSX 6.2.3) or 4789 (default in new installs of NSX 6.2.3 and later) `